src/components/OrdersWidget/OrderRow.tsx

```diff
@@ -94,24 +99,14 @@ interface OrderDetailsProps extends Pick<Props, 'order' | 'pending'> {
 }
 
 const OrderDetails: React.FC<OrderDetailsProps> = ({ buyToken, sellToken, order }) => {
-  const [price, priceInverse] = useMemo((): string[] => {
-    const price = calculatePrice({
-      numerator: { amount: order.priceNumerator, decimals: buyToken.decimals },
-      denominator: { amount: order.priceDenominator, decimals: sellToken.decimals },
-    })
-    const priceInverse = invertPrice(price)
-
-    return [formatPrice(price), formatPrice(priceInverse)]
-  }, [buyToken.decimals, order.priceDenominator, order.priceNumerator, sellToken.decimals])
-
   return (
     <td data-label="Price" className="showResponsive">
       <div className="order-details">
-        <strong>
-          {priceInverse} {displayTokenSymbolOrLink(sellToken)}
-        </strong>
-        <br />
-        {price} {displayTokenSymbolOrLink(buyToken)}
+        <SmartPrice
+          sellToken={sellToken}
+          buyToken={buyToken}
+          price={{ numerator: order.priceNumerator, denominator: order.priceDenominator }}
```

Comment:
This is recreated on each render, so inside SmartPrice `useMemo` deps change every time

Reply:
d49d8beccde88e691bd2c8ae7ed7c92342f5ed50